examples/wakaama/lwm2mconfig.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2018 Beduino Master Projekt - University of Bremen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Configurable parameters for Wakaama example client.
+ *
+ * @author      Christian Manal <manal@uni-bremen.de>
+ *
+ * @}
+ */
+
+
+#ifndef LWM2MCONFIG_H
+#define LWM2MCONFIG_H
+
+/*
+ * RIOT debugging
+ */
+#define ENABLE_DEBUG (1)
+#include "debug.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * DTLS debug log level. One of the following:
+ *    DTLS_LOG_EMERG
+ *    DTLS_LOG_ALERT
+ *    DTLS_LOG_CRIT
+ *    DTLS_LOG_WARN
+ *    DTLS_LOG_NOTICE
+ *    DTLS_LOG_INFO
+ *    DTLS_LOG_DEBUG
+ */
+#ifndef DTLS_LOG_LEVEL
+#define DTLS_LOG_LEVEL DTLS_LOG_CRIT
+#endif
+
+/*
+ * The name the device registers at the LwM2M server.
+ */
+#define LWM2M_DEVICE_NAME "testRIOTDevice"
+
+/*
+ * Device object lifetime on the server.
+ */
+#define LWM2M_DEVICE_TTL 300
+
+/*
+ * LwM2M server URI to register/bootstrap with.
+ * The host part of the URI MUST be a valid IPv6 address. Host names
+ * can not be resolved at this time.
+ */
+#define LWM2M_SERVER_URI "coaps://[fe80::5838:f4ff:fed6:6805]"
+
+/*
+ * Numeric ID of LWM2M_SERVER_URI
+ */
+#define LWM2M_SERVER_ID 10
+
+/*
+ * Set to a true value if LWM2M_SERVER_URI is a bootstrap server.
+ */
+#define LWM2M_SERVER_IS_BOOTSTRAP 0
+
+/*
+ * PSK ID of LWM2M_SERVER_URI.
+ */
+#define LWM2M_SERVER_PSK_ID "fooTestRIOT"
+
+/*
+ * PSK KEY of LWM2M_SERVER_URI in hexadecimal.
+ * Set to NULL to not encrypt traffic.
+ */
+#define LWM2M_SERVER_PSK_KEY "39fe6611deb7713c6069"
```

Comment:
Is this the correct PSK key used by the server given in the README? 
My attempts to make the DTLS handhsake  fails as soon the client key exchange record is sent.

Reply:
As I said, I'm using Eclipse Leshan for testing. You have to set your own keys for each client there in the web interface. This is a key I've generated myself.

Reply:
ah, my bad. I should had read again the README.md before testing the server.